tests/conftest.py

```diff
@@ -203,7 +202,7 @@ def __init__(self, capsys: pytest.CaptureFixture[str]):
         self._screen.set_mode(pyte.modes.LNM)  # \n implies \r
         self._stream = pyte.Stream(self._screen)
 
-    def read_stderr(self) -> list[str]:
+    def read_stderr(self) -> "list[str]":
```

Comment:
maybe just update this file to future annotations?

Reply:
I can send out a separate PR